### ISSUE TITLE
Release May-June

### ISSUE THREAD

**Issue by mattansb** [easystats/easystats#139]
Can we release `performance` and `see` anytime soon?
I am adding the newly supported tests and plots to a vignette in `afex`, which will be released in a few weeks.

**Comment by mattansb** [comment]
Paging maintainer @strengejacke

**Comment by IndrajeetPatil** [comment]
Also `bayestestR` please
https://github.com/easystats/easystats/issues/115#issuecomment-835144305

**Comment by strengejacke** [comment]
I need to submit performance anyway, got a message about warnings in r devel 4.2

**Comment by IndrajeetPatil** [comment]
With the `bayestestR` change, I think we need to update most of the ecosystem and not just a few of them.
 
I think `report` and `modelbased` can go without an update.

**Comment by strengejacke** [comment]
_insight_ will be updated somewhere around mid of june.

**Comment by strengejacke** [comment]
@mattansb What are the changes in _performance_ that are actually important for an urgent update?

**Comment by mattansb** [comment]
Not urgent - can wait until June. Talking about: https://github.com/easystats/parameters/issues/491

**Comment by strengejacke** [comment]
Ok, but indeed I must submit next week, due to CRAN request

**Comment by bwiernik** [comment]
I would just do the cran maintenance fixes

**Comment by mattansb** [comment]
> Ok, but indeed I must submit next week, due to CRAN request

I think that shouldn't be a problem, so long as @DominiqueMakowski doesn't change his mind about which function gets to keep its name (:

**Comment by strengejacke** [comment]
Is there anything that needs to be addressed in performance here before I submit?

**Comment by bwiernik** [comment]
> Is there anything that needs to be addressed in performance here before I submit?

```r
check_best <- function(...) {
  cat("Unzi is the best.")
}
```

![image](https://user-images.githubusercontent.com/4773225/118427751-f4e88f80-b69b-11eb-9df5-cb1453dd816e.jpeg)

**Comment by IndrajeetPatil** [comment]
What a magnificent floof! 😍

**Comment by IndrajeetPatil** [comment]
Release tracker:
- [ ] `performance`
- [ ] `see`
- [ ] `parameters`
- [ ] `effectsize`
- [ ] `bayestestR`

**Comment by strengejacke** [comment]
These two changes in parameters:

* `dispersion` in `model_parameters()` for Bayesian models now defaults
  to `TRUE`.  This change was made to be in line with the default `summary()`
  output from most packages.

* `centrality` in `model_parameters()` for Bayesian models now defaults
  to `"mean"`. This change was made to be in line with the default `summary()`
  output from most packages

break too many packages (though revdep check did not alert any of these?). So I'll revert them for now...

**Comment by bwiernik** [comment]
I can understand reverting for now. But I would like to get both of these in eventually

**Comment by strengejacke** [comment]
Ok, we have to keep some things in mind related to the _parameters_ update:

1. changing defaults for `dispersion` and `centrality` in `model_parameters()` requires updates/changes in packages *correlation*, *effectsize* and *report* (@mattansb @IndrajeetPatil @DominiqueMakowski). Whenever you use `model_parameters()`, please set these arguments explicitly, also in tests!
2. Adding the *Alternative* column in `model_parameters.htest()` requires an _insight_ update. For now, this information is saved as attribute only.
3. Changing the `effects` argument in `model_parameters()` to `"all"` for mixed models (so RE variances are included by default) obviously requires an update from *effectsize* - I think this is already working in the current GitHub version of *effectsize*, right @mattansb ? It also requires an update of *sjPlot*, which will be submitted soon.

So, https://github.com/easystats/parameters/issues/490 and https://github.com/easystats/parameters/issues/496 will be addressed later, earliest after 1. and 2. are resolved and related package updates on CRAN. I think I will suspend the _parameters_ submission until _effectsize_ is updated on CRAN.

**Comment by IndrajeetPatil** [comment]
The `easystats` ecosystem update be like 😅 

![image](https://user-images.githubusercontent.com/11330453/119096562-09b68180-ba14-11eb-95da-cca0b5a12bc3.png)

**Comment by IndrajeetPatil** [comment]
> Whenever you use model_parameters(), please set these arguments explicitly, also in tests!

Noted!

**Comment by bwiernik** [comment]
Looks like there was some disagreement about mean vs median as default. What do folks think?

**Comment by IndrajeetPatil** [comment]
I would vote for sticking to `"median"` as the default: more robust than mean, and we remain backward compatible. 

This is admittedly different from other packages like `brms`, but savvy users can adjust the `centrality` argument in case this inconsistency bothers them.

**Comment by DominiqueMakowski** [comment]
Agreed with @IndrajeetPatil

**Comment by mattansb** [comment]
Also agree

**Comment by DominiqueMakowski** [comment]
(afaik effectsize is good to go on modelbased's side)

**Comment by mattansb** [comment]
Then I'll submit.

**Comment by mattansb** [comment]
effectsize is on its way to CRAN

**Comment by DominiqueMakowski** [comment]
so what's next? insight/bayestestR? Then parameters/modelbased/report?

**Comment by strengejacke** [comment]
Do we really need all those data functions in insight, or can we directly start with our data management package?

**Comment by IndrajeetPatil** [comment]
> we directly start with our data management package

Maybe during the next release cycle? `insight` is not part of this release cycle anyway.

> so what's next?

I thought @strengejacke wanted to submit `parameters` as soon as `effectsize` was accepted.

**Comment by IndrajeetPatil** [comment]
Just received an email from Kurt about `statsExpressions`. I can update it though only after `parameters` is also updated.

@strengejacke Are you planning to submit it soon?

**Comment by strengejacke** [comment]
> Are you planning to submit it soon?

Yes, not sure, if _insight_ first or not... _sjPlot_ is on CRAN, so everything required for _parameters_ update should be finished.

**Comment by mattansb** [comment]
We also need `see` please!

**Comment by strengejacke** [comment]
hm, I can't [_see_](https://github.com/easystats/see/blob/master/NEWS.md) (<- haha) why?

**Comment by strengejacke** [comment]
@DominiqueMakowski is modelbased ready for submission? the _insight_ submission breaks current modelbased on CRAN.

**Comment by DominiqueMakowski** [comment]
it is ready, but it needs latest insight

**Comment by mattansb** [comment]
@strengejacke Can we do a minor version update to see? 🙏

**Comment by strengejacke** [comment]
yes, yes, I just haven't checked if any other packages need to go to CRAN before submitting see? _insight_ is submitted, waiting for feedback, then I planned to submit _parameters_ and _see_.

**Comment by strengejacke** [comment]
But please, @DominiqueMakowski and @mattansb, update the news file :-)

**Comment by mattansb** [comment]
Done (:

**Comment by strengejacke** [comment]
I remember @DominiqueMakowski fixing some stuff on the SEM or Correlation graphs? (https://github.com/easystats/see/issues/95)?

**Comment by DominiqueMakowski** [comment]
https://github.com/easystats/see/blob/master/NEWS.md#changes t'was in the previous version

**Comment by IndrajeetPatil** [comment]
`insight` is on CRAN!

@strengejacke `parameters` submission is still stuck, right?

**Comment by strengejacke** [comment]
@DominiqueMakowski you now need to submit an update for modelbased. I'll submit parameters tomorrow.

**Comment by DominiqueMakowski** [comment]
I'll submit bayestestR first, it's also needed for modelbased

**Comment by strengejacke** [comment]
ok, I needed to disable one test in _modelbased_, because stan_polr models currently don't work after _insight_ update.

**Comment by strengejacke** [comment]
_see_ on CRAN, _parameters_ submitted.

**Comment by IndrajeetPatil** [comment]
`parameters` is also on CRAN!

@strengejacke Can we please bump `insight` version on GitHub? 
Its CRAN version is `0.14.1`, but `GitHub` version is `0.14.0.1`

**Comment by bwiernik** [comment]
I think a good model would be CRAN.9000

**Comment by DominiqueMakowski** [comment]
![](https://media.giphy.com/media/CdY6WueirK8Te/giphy.gif)

**Comment by IndrajeetPatil** [comment]
Now only `report` and `modelbased` are remaining. `correlation` doesn't need to be updated as of yet.

**Comment by DominiqueMakowski** [comment]
as soon as they're up we should strive at pushing datawizard too

**Comment by IndrajeetPatil** [comment]
This cycle is almost complete, save for https://github.com/easystats/modelbased/issues/115 and https://github.com/easystats/report/issues/179.

Closing this issue in favor of those issues.

**Comment by DominiqueMakowski** [comment]
thanks, package modelbased_0.7.0.tar.gz is on its way to CRAN
